R/check_pop_data.R

```diff
@@ -21,13 +21,13 @@ check_pop_data <- function(pop_data) {
   missing_age <- is.element(attributes(pop_data)$age_var, names(pop_data))
 
   if (!missing_age) {
-    stop("Argument `pop_data` is missing column `age` (age, in years).")
+    cli::cli_abort(message = "Argument `pop_data` is missing column `attributes(pop_data)$age_var` (age, in years).")
```

Comment:
this code doesn't evaluate the `attributes()` call; its output is:

![image](https://github.com/UCD-SERG/serocalculator/assets/2474437/6010ed30-8f4e-4cd1-ab80-393d131f763d)

We should use `glue::glue()` or `paste()` here.

Reply:
This has been resolved